### PR TITLE
Updates offline nav doc to note that it's a paid feature

### PR DESCRIPTION
**Pull request by samfader**
Closes https://github.com/mapbox/android-docs/issues/826

### CODE REVIEW

File: src/pages/navigation/overview/offline-routing.md [hunk 1]
```diff
@@ -5,8 +5,14 @@ prependJs:
   - "import { NAVIGATION_VERSION } from '../../../constants';"
   - "import CodeLanguageToggle from '../../../components/code-language-toggle';"
   - "import ToggleableCodeBlock from '../../../components/toggleable-code-block';"
+  - "import Note from '@mapbox/dr-ui/note';"
 ---
 
+{{ <Note title="Downloading offline routing packs" imageComponent={}>
```
Comment (katydecorah) [inline src/pages/navigation/overview/offline-routing.md]: You'll want to pass either an empty div:
```suggestion
{{ <Note title="Downloading offline routing packs" imageComponent={<div/>}>
```

Or `imageComponent={<BookImage width="60" height="60" />}` to have the book image inside the note. You'll also need to `import BookImage from '@mapbox/dr-ui/book-image';` to access that image.

File: src/pages/navigation/overview/offline-routing.md [hunk 2]
```diff
@@ -5,8 +5,14 @@ prependJs:
...
+
+Offline routing pack downloads are restricted to Mapbox Enterprise customers. If you're interested in moving to an Enterprise plan, contact us!
```
Comment (katydecorah) [inline src/pages/navigation/overview/offline-routing.md]: Using HTML inside the note will produce a consistent result:

```suggestion
<p>Offline routing pack downloads are restricted to Mapbox Enterprise customers. If you're interested in moving to an Enterprise plan, contact us!</p>
```

File: src/pages/navigation/overview/offline-routing.md [hunk 3]
```diff
@@ -5,8 +5,14 @@ prependJs:
...
+
```
Comment (katydecorah) [inline src/pages/navigation/overview/offline-routing.md]: Then, you'll want to close the Note component after you're done with the HTML content

```suggestion
</Note>
```

**Comment by samfader** [comment]
@katydecorah and/or @danesfeder - do y'all mind reviewing this?

PS: thanks for your help @katydecorah!

**Pull request merged**